openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -324,6 +324,20 @@ def display_name_with_default(self):
         """
         return course_metadata_utils.display_name_with_default(self)
 
+    @property
+    def display_name_with_default_escaped(self):
```

Comment:
Thanks for confirming.  Yeah, it doesn't require a version bump.  As you say, the changes made in this PR don't actually change the data in the DB, so we're good.